common/src/main/java/bisq/common/persistence/PersistenceManager.java

```diff
@@ -465,7 +465,10 @@ public void writeToDisk(protobuf.PersistableEnvelope serialized, @Nullable Runna
         }
         if (keyRing != null && !keyRing.isUnlocked()) {
             log.warn("Account is not open, ignoring writeToDisk.");
-            UserThread.execute(completeHandler);
+            if (completeHandler != null) {
```

Comment:
We should add a null check a few lines up (line 463) too, no?

Otherwise it's running fine now.

Reply:
Doesn't hurt to have it, added.